.gitlab-ci.yml

```diff
@@ -1,11 +1,30 @@
-netlify:
-  type: deploy
-  script:
-    # Stage
-    - curl -X POST -d {} ${WEBHOOK_STAGE}
-    # Prod
-    - curl -X POST -d {} ${WEBHOOK_PROD}
-    # Dead Man's Snitch Ping
-    - curl ${DMS}
+# This depends on the following env vars to be set in Gitlab:
+#   https://gitlab.com/mozmeao/lumbergh/settings/ci_cd
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+# DMS DeadMansSnitch URL
+# PROD_BUCKET_PATH (w/ s3:// prefix)
+# STAGE_BUCKET_PATH (ex: s3://mozilla-careers-stage)
+
+deploy stage:
+  stage: deploy
+  tags: 
+  - meao
+  variables:
+    BUCKET_PATH: ${STAGE_BUCKET_PATH}
```

Comment:
do the bucket paths have secrets? If not you can add them to this file in a `variables:` directive and they'll be available to all jobs.

Reply:
no, there's nothing secret so I'm switching to `variables`.